coreos-installer

```diff
@@ -572,24 +575,29 @@ get_ignition_url() {
 #########################################################
 mount_tmpfs() {
     log "Mounting tmpfs"
+    if [ -n "$1" ]; then
+        options="remount,size=${1}m"
+    else
+        options="size=${TMPFS_MBSIZE}m"
+    fi
     mkdir -p /mnt/dl
-    mount -t tmpfs -o size=${TMPFS_MBSIZE}m tmpfs /mnt/dl
+    mount -t tmpfs -o "$options" tmpfs /mnt/dl
```

Comment:
nit: It might be a good idea to change `/mnt/dl` into a variable and reuse it. Higher up `imagefile_compressed` and other vars are set under `/mnt/dl`.

Reply:
Agreed.